kratos/tests/test_sympy_fe_utilities.py

```diff
@@ -0,0 +1,87 @@
+import sympy
+
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+import KratosMultiphysics.sympy_fe_utilities as KratosSympy
+
+
+class TestSympyFEUtilities(KratosUnittest.TestCase):
+
+    def test_ScalarOutput(self):
```

Comment:
```suggestion
    def testScalarOutput(self):
```
Very minor naming suggestion.

Reply:
(same in the other tests)